extensions/servlet/test/com/google/inject/servlet/ServletDefinitionPathsTest.java

```diff
@@ -192,6 +198,13 @@ protected void service(HttpServletRequest servletRequest,
         .andReturn(servletPath)
         .anyTimes();
 
+    // computePath()
+    if(servletPath.isEmpty()) {
+        expect(request.getRequestURI()).andReturn(requestUri);
+        expect(request.getContextPath())
+        .andReturn(contextPath);
```

Comment:
(this is an example of where you'd use a 4 space indent since it's a continuation)